### PR TITLE
Fix saveLayer calls

### PR DESCRIPTION
**Pull request by chrishtr**
A commit [a few weeks ago](https://github.com/browserengineering/book/commit/91a0762c243e471f8e033d1743b1fe482931fd02) changed `saveLayer(paint=my_paint)` to `saveLayer(my_paint`), which is wrong because `paint` is the second parameter. This caused browse15.py to crash loading our website.

### CODE REVIEW

File: book/visual-effects.md [hunk 1]
```diff
@@ -1075,7 +1075,7 @@ Here, the `saveLayer` call asks Skia[^layer-surface] to draw all the
 children to a separate
 surface before blending them into the parent once
 `restore` is called.
-The `paint` option to `saveLayer` specifies the specific type of
+The secondc parameter to `saveLayer` specifies the specific type of
```
Comment (pavpanchekha) [inline book/visual-effects.md:1078]: Typo

File: book/visual-effects.md [hunk 2]
```diff
@@ -1385,7 +1385,7 @@ class Blend:
         paint = skia.Paint(
             BlendMode=parse_blend_mode(self.blend_mode),
         )
-        canvas.saveLayer(paint)
+        canvas.saveLayer(none, paint)
```
Comment (pavpanchekha) [inline book/visual-effects.md:1388]: None

**Review by pavpanchekha — APPROVED** [review]

File: book/visual-effects.md (hunk 1 above)
Comment (chrishtr) [inline book/visual-effects.md:1078]:
```suggestion
The second parameter to `saveLayer` specifies the specific type of
```

File: book/visual-effects.md (hunk 2 above)
Comment (chrishtr) [inline book/visual-effects.md:1388]:
```suggestion
        canvas.saveLayer(None, paint)
```

**Pull request merged**